src/vizcircuit.jl

```diff
@@ -0,0 +1,159 @@
+using Viznet: canvas
+using YaoBlocks
+
+export CircuitStyles, CircuitGrid, circuit_canvas, vizcircuit
+
+module CircuitStyles
+	using Compose
+    r = 0.2
```

Comment:
why these are not `const Ref`? the canonical way of implementing global tweakable options is to use e.g

```jl
const r = Ref{Float64}(0.2)
```